func main() {
  var a;
  a = foo(5);
  print("should not crash yet");
  print(x);
}

/*
*OUT*
should not crash yet
ErrorType.NAME_ERROR
*OUT*
*/